src/main/java/it/gov/pagopa/paymentupdater/service/PaymentService.java

```diff
@@ -14,7 +14,7 @@ public interface PaymentService {
 
 	void save(Payment reminder);
 
-	Map<String, Boolean> checkPayment(Payment payment) throws JsonProcessingException, InterruptedException, ExecutionException;
+	Map<String, String> checkPayment(Payment payment) throws JsonProcessingException, InterruptedException, ExecutionException;
```

Comment:
Ditto for return type

Reply:
This has been fixed in the develop-rptid branch that will be merged later